agot-bg-game-server/src/client/game-state-panel/ChooseHouseCardComponent.tsx

```diff
@@ -14,6 +14,24 @@ import { observable } from "mobx";
 @observer
 export default class ChooseHouseCardComponent extends Component<GameStateComponentProps<ChooseHouseCardGameState>> {
     @observable selectedHouseCard: HouseCard | null;
+    @observable chosenHouseCard: HouseCard | null;
```

Comment:
Why is this field needed? Couldn't you get the chosen house card of the player in the GameState, like it is done in `BiddingComponent` ?

Reply:
Yea of course. I change that.

Reply:
Ah I guess I added because in live games you don't get your own house card. Server always sends `null` to all clients. Should we change the implementation to broadcast `null` to all others but the chosen house card to their owners?

Reply:
>Ah I guess I added this field because in live games you don't get your own house card. Server always sends null to all clients.

Are you sure? When I read the code, it feels to me that the server will send the correct house card to the client.

Reply:
But `serializeToClients` isn't called in a live game is it. Game reacts on `onServerMessage`, doesn't it?

Reply:
Oh, ok, right, I didn't know you were talking about this part of the code. Then, you can quickly modify this part of the code so that the server sends the house card.

Otherwise, I'll do it in an other PR.

Reply:
I‘ll do it later:)

Reply:
Done. :)